src/shared/scrapers/USA/CA/sonoma-county.js

```diff
@@ -14,6 +15,9 @@ const scraper = {
   async scraper() {
     const $ = await fetch.page(this.url);
     const $th = $('th:contains("Total in Sonoma County")');
+    if ($th.html() === null) {
+      throw new DeprecatedError('Could not find table');
+    }
```

Comment:
```suggestion
```

Reply:
Yeah @chunder I'd say this one should just fail, but we absolutely can throw a normal error here `throw new Error('Sonoma has a ArcGIS dashbaord we need to scrape')`